modules/consensus/accept_test.go

```diff
@@ -1349,7 +1351,7 @@ func (cst *consensusSetTester) testPaymentChannelBlocks() error {
 		// txn needs to be fully custom. To get a custom txn, manually create an
 		// address and then use the wallet to fund that address.
 		channelSize := types.NewCurrency64(10e3)
-		channelFundingSK, channelFundingPK, err := crypto.GenerateSignatureKeys()
```

Comment:
I'd actually prefer to avoid this if possible.

I eventually want to switch all the `skg` objects to `KeyPairGenerator` interfaces so that we break up tight coupling between `SignatureKeyGenerator` and its clients (and can therefore test its clients more easily by mocking out `KeyPairGenerator`).

It's not a huge deal because the code delta between the two solutions is small, but I feel like the current code is a step closer to where we want to be. I can change it to your suggestion if this is something you feel strongly about.